Index: src/use_case/TopSongs/TopSongsInputData.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package use_case.TopSongs;\nimport entity.User;\n\npublic class TopSongsInputData {\n    private String timeframe;\n\n    public TopSongsInputData(String timeframe){\n        this.timeframe = timeframe;\n    }\n\n    public void setTimeframe(String timeframe) {\n        this.timeframe = timeframe;\n    }\n\n    public String getTimeframe() {\n        return timeframe;\n    }\n}\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/use_case/TopSongs/TopSongsInputData.java b/src/use_case/TopSongs/TopSongsInputData.java
--- a/src/use_case/TopSongs/TopSongsInputData.java	
+++ b/src/use_case/TopSongs/TopSongsInputData.java	
@@ -1,11 +1,12 @@
 package use_case.TopSongs;
-import entity.User;
 
 public class TopSongsInputData {
     private String timeframe;
+    private final String id;
 
-    public TopSongsInputData(String timeframe){
+    public TopSongsInputData(String timeframe, String id){
         this.timeframe = timeframe;
+        this.id = id;
     }
 
     public void setTimeframe(String timeframe) {
@@ -15,4 +16,8 @@
     public String getTimeframe() {
         return timeframe;
     }
+
+    public String getId(){
+        return id;
+    }
 }
